internal/uvm/update_uvm.go

```diff
@@ -4,10 +4,12 @@ package uvm
 
 import (
 	"context"
+	"fmt"
 
 	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
 	"github.com/Microsoft/hcsshim/pkg/annotations"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/Microsoft/hcsshim/pkg/ctrdtaskapi"
+	"github.com/opencontainers/runtime-spec/specs-go"
 )
 
 func (uvm *UtilityVM) UpdateConstraints(ctx context.Context, data interface{}, annots map[string]string) error {
```

Comment:
`UpdateConstraints` doesn't seem like the right name for this function if it's also used to pass through fragments. Suggest either renaming or moving the fragment code out (whichever makes more sense).